doc/nrf/gs_installing.rst

```diff
@@ -229,9 +229,9 @@ To clone the repositories, complete the following steps:
 #. Determine what revision of the |NCS| you want to work with.
    The recommended way is to work with a specific release.
 
-   * To work with a specific release, the revision is the corresponding tag (for example, ``v1.2.0``).
+   * To work with a specific release, the revision is the corresponding tag (for example, ``v1.3.0``).
      You can find the tag in the :ref:`release_notes` of the release.
-   * To work with a development tag, the revision is the corresponding tag (for example, ``v1.2.99-dev1``)
+   * To work with a development tag, the revision is the corresponding tag (for example, ``v1.3.99-dev1``)
```

Comment:
is this change ok @ru-fu ?

Reply:
I think it might be clearer to leave it at a tag that actually exists - what do you think, @carlescufi ?

Reply:
yes, please leave the existing one

Reply:
reverted to v1.2.99-dev1